pytorch_lightning/trainer/callback_hook.py

```diff
@@ -21,6 +21,16 @@ def on_init_end(self):
         for callback in self.callbacks:
             callback.on_init_end(self)
 
+    def on_fit_start(self):
+        """Called when the trainer initialization begins, model has not yet been set."""
+        for callback in self.callbacks:
```

Comment:
it would be nice to have Callback collection to remove this loop
https://github.com/PyTorchLightning/pytorch-lightning/blob/88f816ed06ed4e39dcb79d53abb5fe0613f0d36f/pytorch_lightning/loggers/base.py#L251